src/mono/wasm/runtime/driver.c

```diff
@@ -1323,3 +1336,13 @@ mono_wasm_load_profiler_aot (const char *desc)
 }
 
 #endif
+
+static void
+mono_wasm_init_finalizer_thread (void)
+{
+	// At this time we don't use a dedicated thread for finalization even if threading is enabled.
+	// Finalizers periodically run on the main thread
+#if 0
```

Comment:
if possible let's replace these finalizer related 'if 0's with a common name that we plan to define eventually, so anyone looking for the rest of them can just grep the tree for it instead of having to examine every if 0

Reply:
Do you think it's even worth keeping?  I have a feeling a lot of wasm-targeting code will want to do finalization on the main thread only, so we might never enable this.